dotcom-rendering/src/experiments/tests/auxia-sign-in-gate.js

```diff
@@ -0,0 +1,23 @@
+export const auxiaSignInGate = {
+	id: 'AuxiaSignInGate',
+	start: '2025-01-23',
+	expiry: '2026-01-30',
+	author: 'Pascal (Growth Team)',
+	description:
+		'R&D Experiment: using Auxia API to drive the behavior of the SignIn gate',
+	audience: 0.01,
+	audienceOffset: 0,
+	successMeasure: '',
+	audienceCriteria: '',
+	ophanComponentId: 'auxia_signin_gate',
+	dataLinkNames: 'AuxiaSignInGate',
+	idealOutcome: '',
+	showForSensitive: false,
+	canRun: () => true,
+	variants: [
+		{
+			id: 'auxia-signin-gate',
+			test: () => {},
+		},
+	],
```

Comment:
Depending on how you've agreed to do this with your data analysts, we [usually define a variant](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/experiments/tests/optimise-spacefinder-inline.ts#L13-L26) (called `variant`) and a control group here. I believe this makes it easy for data analysts to filter and cut the data. 

@domlander can I pick on you for confirmation of this?

Reply:
We are not yet in the process of analysis. It's just a proof of concept. For the moment it's more a feature than an ab test :)

Reply:
Okay, thanks @shtukas. Are you proving that the integration works to your satisfaction? In which case, would a 0% test be better? This would allow you to force yourself into the test to make sure it was working correctly, rather than impacting real users.

Reply:
Actually we want to impact real users, but I guess that as a preliminary stage I can set it to zero and increase later. Give me one sec....

Reply:
Done!